diff --git a/CMake/config.cmake b/CMake/config.cmake
index 65a023e..ce077cb 100644
--- a/CMake/config.cmake
+++ b/CMake/config.cmake
@@ -134,5 +134,5 @@ if (BUILD_INSTALLER)
 
     install(FILES "${LICENSE_FILE}" "${README_FILE}"
             DESTINATION "."
-            PERMISSIONS OWNER_READ)
+            PERMISSIONS OWNER_READ GROUP_READ WORLD_READ)
 endif()
diff --git a/CMake/utility.cmake b/CMake/utility.cmake
index f12b576..582aad7 100644
--- a/CMake/utility.cmake
+++ b/CMake/utility.cmake
@@ -165,7 +165,7 @@ function(loadFolder FOLDER _HEADER_FILES _SOURCE_FILES)
     getNameOfDir(CMAKE_CURRENT_SOURCE_DIR DIRNAME)
     if (${shouldInstall})
       if (NOT ${FULL_HEADER_PATH} MATCHES ".*_p.h$") # we don't want to install header files which are marked as private
-        install(FILES ${FULL_HEADER_PATH} DESTINATION "include/${DIRNAME}/${REL_PATH}" PERMISSIONS OWNER_READ)
+        install(FILES ${FULL_HEADER_PATH} DESTINATION "include/${DIRNAME}/${REL_PATH}" PERMISSIONS OWNER_READ GROUP_READ WORLD_READ)
       endif()
     endif()
   endforeach()
diff --git a/doc/CMakeLists.txt b/doc/CMakeLists.txt
index 4a46cea..88fd95e 100644
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -33,7 +33,7 @@ set(DOXYGEN_PROJECT_NAME      "${PROJECT_NAME}")
 set(DOXYGEN_INPUT_DIRECTORY   "${CMAKE_SOURCE_DIR}/src/rttr \\
                                ${CMAKE_CURRENT_SOURCE_DIR}" )
 set(DOXYGEN_OUTPUT_DIRECTORY  "${CMAKE_BINARY_DIR}/doc")
-set(DOXYGEN_DOC_INSTALL_DIR   "doc")
+set(DOXYGEN_DOC_INSTALL_DIR   "share/doc/rttr-git")
 set(DOXYGEN_CONFIG_FILE_IN    "DoxyFile.cfg.in")
 set(DOXYGEN_LAYOUT_FILE_IN    "doxygen-customization/layout/DoxygenLayout.xml.in")
 set(DOXYGEN_EXCLUDE           "")
diff --git a/src/rttr/CMakeLists.txt b/src/rttr/CMakeLists.txt
index c21f7ed..6e80276 100644
--- a/src/rttr/CMakeLists.txt
+++ b/src/rttr/CMakeLists.txt
@@ -58,7 +58,7 @@ if (${BUILD_RTTR_DYNAMIC})
     if (BUILD_INSTALLER)
         install(TARGETS rttr_core EXPORT rttr_targets
                 RUNTIME DESTINATION ${RTTR_BIN_INSTALL_DIR}
-                LIBRARY DESTINATION ${RTTR_BIN_INSTALL_DIR}
+                LIBRARY DESTINATION ${RTTR_LIB_INSTALL_DIR}
                 ARCHIVE DESTINATION ${RTTR_LIB_INSTALL_DIR})
 
         install(EXPORT rttr_targets
